src/types.rs

```diff
@@ -1,10 +1,21 @@
 use anyhow::{bail, Result};
-pub type Address = String;
-pub type MassaHash = String;
-pub type Signature = String;
-pub type PublicKey = String;
+use serde::{de::DeserializeOwned, Serialize};
+
+/// How we choose to represent our identifier everywhere
+pub type Hash = String;
```

Comment:
The issue with that is that we would prefer to use an `&str` rather than a `String` (without dealing with explicit lifetime required by type aliases, like having and `pub type Hash<'a> = &'a str;` ...)